test/e2e_node/container.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"errors"
+
+	"k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+//One pod one container
+type ConformanceContainer struct {
+	Container api.Container
+	Client    *client.Client
+}
+
+func NewConformanceContainer(cl *client.Client, container api.Container) ConformanceContainer {
+	return ConformanceContainer{
+		Container: container,
+		Client:    cl,
+	}
+}
+
+func createContainer(cl *client.Client, container api.Container) error {
```

Comment:
I prefer to use 'createContainer' since I want to test container, using pod is just the underling implementation way. Since new code will use method for struct, so it will just be 'Create'.